src/page-modules/departures/details/index.tsx

```diff
@@ -49,6 +49,10 @@ export function DeparturesDetails({
     .map((s) => s.situationNumber)
     .filter((s): s is string => !!s);
 
+  const isJourneyCancelled = situations.some((item) =>
+    item.summary.some((i) => i.value === 'Journey cancelled'),
```

Comment:
Is this the only way to know if an entire journey is cancelled? Just feels to me a bit fragile checking a string instead of a property like the estimated calls have. 

And if so, shouldn't the `situations` for the service journey be checked instead of the estimated call? (might not matter which one is checked since they both will have "Journey cancelled" possibly?)

Reply:
I agree that the solution is a bit fragile. I can search a bit further but I haven't found a better solution yet.

Reply:
I am viewing a departure that was cancelled earlier today at 06.45. The prop focusedCall.cancellation : false. Should´t this prop be true, since the departure was cancelled?    
  
<img width="712" alt="Screenshot 2023-12-20 at 14 19 11" src="https://github.com/AtB-AS/planner-web/assets/59939294/f89605dd-19fd-4eee-9744-3aca8cc56fb1">

Reply:
I would assume so 🤔 Actual departure time is null as well, which should be updated with the actual departure time after it has passed a stop, so I guess it is true that this was cancelled. Not sure why the cancellation is false though

Reply:
Interesting. I also checked a departure that was carried out as usual and both actualArrivalTime and actualDepartureTime are set to null here also.

<img width="703" alt="Screenshot 2023-12-20 at 14 43 43" src="https://github.com/AtB-AS/planner-web/assets/59939294/424292e2-77bd-4568-ab48-443600c5f9d8">

Reply:
Tried some routes that have been canceled recently. Here cancellation is set to true. Is there a difference between two-digit routes and three-digit routes when it comes to reporting? 

<img width="511" alt="Screenshot 2023-12-20 at 16 28 12" src="https://github.com/AtB-AS/planner-web/assets/59939294/f2773e5e-dc52-47f2-af57-364debb317ec">
<img width="518" alt="Screenshot 2023-12-20 at 16 28 26" src="https://github.com/AtB-AS/planner-web/assets/59939294/8211bd33-5bb6-489e-8a7c-dfa8bb24f03f">

Reply:
Anyway, I´ll update the condition to check for the cancellation prop. Exploring whether cancellation in some cases is not set when it should is a separate task that in my opinion should be solved in a separate branch.